install-dev/install_version.php

```diff
@@ -25,8 +25,8 @@
  */
 
 define('_PS_INSTALL_VERSION_', '1.7.8.0');
-define('_PS_INSTALL_MINIMUM_PHP_VERSION_ID_', 70103);
-define('_PS_INSTALL_MAXIMUM_PHP_VERSION_ID_', 70499);
+define('_PS_INSTALL_MINIMUM_PHP_VERSION_ID_', 70205);
+define('_PS_INSTALL_MAXIMUM_PHP_VERSION_ID_', 80099);
 
-define('_PS_INSTALL_MINIMUM_PHP_VERSION_', '7.1.3');
-define('_PS_INSTALL_MAXIMUM_PHP_VERSION_', '7.4');
+define('_PS_INSTALL_MINIMUM_PHP_VERSION_', '7.2.5');
```

Comment:
Breaking change to be documented. Are there others?

Reply:
This Breaking change was already documented on the Symfony 4.4 PR ;)